### PR TITLE
add minikube

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

### CODE REVIEW

**Review by rawlingsj — DISMISSED** [review]

**Comment by developer-guy** [comment]
I don't know which packages do we need at runtime so I didn't add anything, can you help us to provide some guidance @dlorenc 🫣

File: minikube.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - busybox
+      - go
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/kubernetes/minikube/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 90fd915150fc69221ed9824bdbd59974bede1470bff8144145fcfadea8a8d27f
+  - runs: |
+      make out/minikube-linux-${{build.arch}}
+      install -D out/minikube-linux-${{build.arch}} ${{targets.destdir}}/bin/minikube
```
Comment (rawlingsj) [inline minikube.yaml:28]:
```suggestion
      install -D out/minikube-linux-${{build.arch}} ${{targets.destdir}}/usr/bin/minikube
```

**Comment by rawlingsj** [comment]
I just did a local build with the suggested change above then installed it with the wolfi sdk, I see

```
[sdk] ❯ minikube start
* minikube v1.29.0 on Wolfi  (docker/arm64)
* Unable to pick a default driver. Here is what was considered, in preference order:
* Alternatively you could install one of these drivers:
  - docker: Not installed: exec: "docker": executable file not found in $PATH
  - kvm2: Not installed: exec: "virsh": executable file not found in $PATH
  - podman: Not installed: exec: "podman": executable file not found in $PATH
  - qemu2: Not installed: exec: "qemu-system-aarch64": executable file not found in $PATH
  - vmware: Not installed: exec: "docker-machine-driver-vmware": executable file not found in $PATH
  - virtualbox: Not installed: unable to find VBoxManage in $PATH

X Exiting due to DRV_NOT_DETECTED: No possible driver was detected. Try specifying --driver, or see https://minikube.sigs.k8s.io/docs/start/

```

**Comment by rawlingsj** [comment]
What's the intention with minikube, is it to run with docker, mounting the socket for example?  I wonder if this something we want to do given the issues with mounting a docker socket?

**Comment by rawlingsj** [comment]
I had a slack chat with @developer-guy who was wandering about adding qemu as a package to wolfi.  Is that a good / bad idea?  I wasn't sure.

I noticed alpine has the [package ](https://git.alpinelinux.org/aports/tree/community/qemu/APKBUILD).  I'm not 100% sure if this makes sense so putting it out there.  There's a few dependencies that would also need to be built too.

**Comment by imjasonh** [comment]
Closing this, please feel free to reopen if you want to move this forward.

**Pull request closed without merging**